Update Wiki conversion docs to replace INFRA-2328 text (#3268)
jenkins-infra/jenkins.io: CONTRIBUTING.adoc

. Click the _Convert_ button and wait till the files are generated
. Place the Asciidoc content and the images in the correct repository locations
. Review nd correct the exported file formatting
** Remove macro references in the top of the document
** Remove any "Table of contents" or replace it by macros if needed
** Repair converted code blocks from Pandoc generated tables to Asciidoc code blocks
. Review the content
** Replace any `todo-replace-by-actual-path` image paths in the exported pages with the actual images directory path (`/images/...`)
** Correct flawed terminology and outdated naming.  For example, we use "agent" rather than "slave" and "pipeline" instead of "workflow")
. Commit changes, push them to your fork and create a pull request against link:https://github.com/jenkins-infra/jenkins.io[the repository]
. Once the pull request is merged, submit a pull request to the
  link:https://github.com/jenkins-infra/jenkins-infra/blob/staging/dist/profile/templates/confluence/vhost.conf[`vhost.conf`] file in the
  link:https://github.com/jenkins-infra/jenkins-infra[`jenkins-infra/jenkins-infra`] repository to redirect from `wiki.jenkins.io` to `jenkins.io`.

>   The pull request will usually add to either the "NOn plugin rewrites" section or the "Developer documentation section, depending on the content of the redirected page +

Reviewer: ```suggestion
  The pull request will usually add to a subsection under the "Non plugin rewrites" section, depending on the content of the redirected page +
```

There are multiple sections there
